Cue/app/tabs/account/AccountHome.js

```diff
@@ -87,7 +88,7 @@ class AccountHome extends React.Component {
       }).catch(e => {
         console.warn('Failed to sync changes', e)
         Alert.alert(
-          'Some changes haven’t been synced to the Cue cloud yet',
+          'Some Changes Haven’t Been Synced to the Cue Cloud Yet',
```

Comment:
Are we just going to title case everything now? Or keep the android/ios versions